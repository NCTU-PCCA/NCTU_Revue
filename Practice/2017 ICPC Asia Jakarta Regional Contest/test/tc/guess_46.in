229753876215296383 58
<<<<<<<><>>><<<<>><><>><<>>>><<<><<<><<<<<<<<<>>>>>><<>><>
