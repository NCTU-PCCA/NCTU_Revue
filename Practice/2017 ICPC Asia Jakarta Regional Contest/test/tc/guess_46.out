229753876215296383 229753876215296382 229753876215296381 229753876215296380 229753876215296379 229753876215296378 229753876215296377 1 229753876215296376 2 3 4 229753876215296375 229753876215296374 229753876215296373 229753876215296372 5 6 229753876215296371 7 229753876215296370 8 9 229753876215296369 229753876215296368 10 11 12 13 229753876215296367 229753876215296366 229753876215296365 14 229753876215296364 229753876215296363 229753876215296362 15 229753876215296361 229753876215296360 229753876215296359 229753876215296358 229753876215296357 229753876215296356 229753876215296355 229753876215296354 229753876215296353 16 17 18 19 20 21 229753876215296352 229753876215296351 22 23 229753876215296350 24
